docs/technical-documentation.md

```diff
@@ -7,50 +7,50 @@ category: Technical documentation
 
 ## Micro-services architecture
 
-Here is an overview of PowSyBl micro-services architecture and corresponding repositories:
+Here is an overview of PowSyBl's microservice architecture and corresponding repositories:
 
 <object type="image/svg+xml" data="./architecture/images/micro-services-architecture.svg" >
 </object>
 
 ### Case server
 
-This service is responsible from raw network data storage. It relies on a file system to store its data and allows uploading and fetching network for any format supported by PowSyBl (CGMES, UCTE, XIIDM, etc).
+This service is responsible for raw network data storage. It relies on a file system to store its data and allows uploading and fetching networks in any format supported by PowSyBl (CGMES, UCTE, XIIDM, etc).
 
 ### Network store server
 
-Almost all of the feature developed in PowSyBl relies on IIDM API for accessing network data, the idea of the service is to re-implement the IIDM API to be better integrated in a micro-services typical architecture. Instead of an in-memory implementation like in PowSyBl core repository, this implementation is backed by a Cassandra database. Network model is stored in the database in a detailed way (one table per equipment), so that we can query only the amount of needed data and ensure a good performance for a simple switch position change (few data), a single line diagram rendering (substation wide data) or a loadflow run (network wide data). Network data should only be accessed from this REST Web service using the provided Java client (the one which implement IIDM API) and never directly from the REST interface.
+Almost all the features developed in PowSyBl rely on the IIDM API for accessing network data. The idea of this service is to re-implement the IIDM API to be better integrated in a typical microservice architecture. Instead of an in-memory implementation like in the PowSyBl core repository, this implementation is backed by a Cassandra database. The network model is stored in the database in a structured way (one table per equipment type and with indexes), so that we can query only the needed data and ensure good performance for common operations. Examples are a simple switch position change (only a few values), a single line diagram rendering (substation wide data) or a loadflow run (network wide data). The REST interface exposed by this service is very low-level for performance reasons. In particular, the write operations do not prevent inconsistent modifications. Therefore, network data should usually be accessed from this service using the provided Java client (the one which implements the IIDM API) and not directly from the REST interface because the java client is safe and prevents inconsistent modifications.
```

Comment:
> Examples are a simple switch position change (only a few values), a single line diagram rendering (substation wide data) or a loadflow run (network wide data).

It was not very clear for me at first reading that those were examples of the aforementioned "common operations". Maybe "Examples of the latter"?

Reply:
I used "Examples of common operations"